src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery__list');
+const bigPicture = document.querySelector('.gallery__large-img');
+
+gallery.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target.closest('.list-item__link');
+
+  if (!target) {
+    return;
+  }
+
+  bigPicture.src = target.href;
```

Comment:
There is an issue here: `target.href` might not be the correct way to get the image URL if the link is not directly pointing to an image. Ensure that `target.href` contains the correct URL for the image you want to display in `bigPicture.src`.